tests/block-period-checkout.test.js

```
@@ -230,4 +239,41 @@ describe("block period checkout and availability rules", () => {
   });
+
+  it("returns insufficient-lead-time from checkWindowAvailability", async () => {
+    const bookable = blockPeriodBookable({
+      preparationLeadTimeMinutes: 120,
+      serviceHours: SERVICE_HOURS,
+    });
+    const instance = getWeekendInstance();
+    const provider = buildProvider(bookable);
+    const originalMembershipLookup =
+      MembershipManager.getMembershipsByTenantAndRoles;
+    const clock = sinon.useFakeTimers(localDate("2026-06-05", "17:30").getTime());
```

ESLint

[error] Replace `localDate("2026-06-05",·"17:30").getTime()` with `⏎······localDate("2026-06-05",·"17:30").getTime(),⏎····`

(prettier/prettier)